database/migrations/20200715123925_update_withdrawal_history_exchange/postgres.sql

```diff
@@ -0,0 +1,10 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+ALTER TABLE withdrawal_history RENAME COLUMN exchange TO exchange_name_id;
+ALTER TABLE withdrawal_history ALTER COLUMN exchange_name_id SET DATA TYPE UUID USING (gen_random_uuid());
+ALTER TABLE withdrawal_history ADD CONSTRAINT fk_exchange_id_withdrawal_history FOREIGN KEY(exchange_name_id) REFERENCES exchange(id);
```

Comment:
I have made updates to the schema for both sqlite and postgres 

It will now check if any existing records in the withdrawal_history table exist and create an exchange record for it and update can you please test again